packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -5,9 +5,17 @@
 import React from 'react';
 import AppLayout from '../AppLayout';
 import AppErrorBoundary from '@fxa-components/AppErrorBoundary';
+import { QueryParams } from '../../lib/types';
+import FlowEvents from '../../lib/flow-event';
 import './index.scss';
 
-export const App = () => {
+type AppProps = {
+  queryParams: QueryParams;
+};
+
+export const App = ({ queryParams }: AppProps) => {
+  FlowEvents.init(queryParams, true);
```

Comment:
If that second arg is always true then we could remove the param from `FlowEvent.init`.

Reply:
In an ideal world this would be shared between Payments and Settings, which is why I had it as an option. But I don't think that's going to happen this PR, so I'll remove it.

Reply:
Can you file an issue (maybe outside of the Settings epic) for that?  I want that as well.

Reply:
Yes!